@xen-orchestra/backups/_ContinuousReplicationWriter.js

```diff
@@ -116,3 +115,5 @@ export class ContinuousReplicationWriter {
     }
   }
 }
+
+exports.ContinuousReplicationWriter = ContinuousReplicationWriter
```

Comment:
Why not exporting it directly?

```js
exports.ContinuousReplicationWrite = class ContinuousReplicationWrite {
```

Reply:
I  did this type of export for all modules. And , this type of practice is done before in the library.

Reply:
Less code, less repetition, closer to ES modules.